src/core/ext/xds/xds_client.cc

```diff
@@ -66,6 +66,7 @@
 namespace grpc_core {
 
 TraceFlag grpc_xds_client_trace(false, "xds_client");
+TraceFlag grpc_xds_client_ref_count_trace(false, "xds_client_ref_count");
```

Comment:
Please change "ref_count" to "refcount", since this will allow this tracer to be enabled/disabled all at once with the other ref-count tracers.  For context, see:

https://github.com/grpc/grpc/blob/0dd3f7e65b4938de736ccd0326aa26523df4649c/src/core/lib/debug/trace.cc#L51

(In https://github.com/grpc/grpc/blob/master/doc/environment_variables.md, we document the fact that you can enable all refcount tracers with "refcount".)

Reply:
I see!  Fixed.  PTAL thank you!